configure.py

```diff
@@ -26,8 +26,6 @@
 import sys
 import logging
 
-_DEFAULT_CUDA_VERISON = "10.1"
-_DEFAULT_CUDNN_VERSION = "7"
```

Comment:
Very much like the new modular functions, but I'm wondering if removing these 2 from global actually hurts the readability / maintainability. I'm thinking when we update to 10.2+ into the future it's logical to just change variables at the top of the file instead of reading through the whole code to find variables that need updated.

If you have a strong preference here, I'm not tied to that opinion, but my thoughts.

Reply:
Mhhh. True. The best would actually be to fetch those information from tensorflow and not ask the user at all. But I don't know how to do that.